pygments/lexers/configs.py

```diff
@@ -45,12 +45,21 @@ class IniLexer(RegexLexer):
         'root': [
             (r'\s+', Whitespace),
             (r'[;#].*', Comment.Single),
-            (r'\[.*?\]$', Keyword),
-            (r'(.*?)([ \t]*)(=)([ \t]*)([^\t\n]*)',
+            (r'(\[.*?\])([ \t]*)$', bygroups(Keyword, Whitespace)),
+            (r'(.*?)([  \t]*)([=:])([  \t]*)([^;#\n]*)(\\)(\s+)',
+             bygroups(Name.Attribute, Whitespace, Operator, Whitespace, String, Text, Whitespace),
+             "value"),
+            (r'(.*?)([  \t]*)([=:])([  \t]*)([^ ;#\n]*(?: +[^ ;#\n]+)*)',
```

Comment:
Hi, thanks for your review.
I saw that a huge work has been done to catch non significant white spaces. So the regex on line 52 will not catch the space(s) placed between the string value and a possible inline comment. Otherwise that or these spaces would be catched as part of the value string.
Test case : `foo = bar foo # this is an inline comment`.